lib/beaker/host/cisco.rb

```diff
@@ -95,7 +95,7 @@ def environment_string env
       env_array = self.environment_variable_string_pair_array( env )
       environment_string = env_array.join(' ')
 
-      command = self[:platform] =~ /cisco-5/ ? 'export' : 'env'
+      command = self[:platform] =~ /cisco-nexus/ ? 'export' : 'env'
```

Comment:
We have test that will set the environment prior to executing a command. On Nexus, we found the using the 'env' command didn't work for this, we had to use the export command.